src/output_widget.cc

```diff
@@ -0,0 +1,73 @@
+#include "output_widget.h"
+
+#include <gtkmm/textview.h>
+
+#include <memory>
+
+namespace bb
+{
+    class OutputWidget::Impl
+    {
+    public:
+        Impl();
+        void set_line(int line, const std::string &text);
+        void create_empty_lines(int n);
+        void delete_line(int line);
+
+        Gtk::TextView tv;
+        Glib::RefPtr<Gtk::TextBuffer> buffer;
```

Comment:
I understand that you use this member for simplicity, but in this case and not needing the ownership of the object, it results in a little bit overkill because you are increasing the reference counting. A better way to do that is to use a raw pointer.

```suggestion
        Gtk::TextBuffer *buffer;
``

Reply:
fixed! 0b43538